tests/rate_test.go

```diff
@@ -20,3 +20,27 @@ func (c *ClientTests) TestRateRetrieve() {
 	assert.Equal(reflect.TypeOf(&easypost.Rate{}), reflect.TypeOf(rate))
 	assert.True(strings.HasPrefix(rate.ID, "rate_"))
 }
+
+func (c *ClientTests) TestBetaStatelessRateRetrieve() {
+	client := c.TestClient()
+	assert, require := c.Assert(), c.Require()
+
+	rates, err := client.BetaGetStatelessRates(c.fixture.BasicShipment())
+	require.NoError(err)
+
+	assert.Equal(reflect.TypeOf([]*easypost.StatelessRate{}), reflect.TypeOf(rates))
+}
+
+func (c *ClientTests) TestBetaStatelessRateGetLowest() {
```

Comment:
If it's easy enough, please also test the error thrown when no rates can be found due to bad carrier/service like we do with the other lowest rate tests